lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -347,357 +280,387 @@
+                @include margin-right(1px);
+              }
 
-      // ====================
+              &:before {
+                border-color: $transparent;
+                border-bottom-color: $gray-l3;
+                border-width: 7px;
+              }
+            }
 
-      // CASE: "enrolled as" status - professional ed
-      &.professional {
+            .actions-item {
+              @extend %t-title7;
+              display: block;
+              margin: 0;
 
-        // changes to cover
-        .cover {
-          border-color: $professional-color-lvl3;
-          padding: ($baseline/10);
-        }
+              &.is-hidden {
+                display: none;
+              }
 
-        // course enrollment status message
-        .sts-enrollment {
-          .label {
-            @extend %text-sr;
-          }
+              .action {
+                @include margin-right(0);
 
-          // status message
-          .sts-enrollment-value {
-            background: $professional-color-lvl3;
+                &:hover, &:focus {
+                  border: 1px solid transparent;
+                 }
+              }
+            }
           }
         }
       }
 
-      // CASE: "enrolled as" status - verified
-      &.verified {
+      .course-status {
+        background: $yellow;
+        border: 1px solid $border-color-2;
+        box-shadow: 0 1px 0 0 rgba(255,255,255, 0.6);
+        margin-top: 17px;
+        margin-right: flex-gutter();
+        padding: ($baseline/4);
+        width: flex-grid(8);
+        float: left;
+        @include box-sizing(border-box);
 
-        // changes to cover
-        .cover {
-          border-color: $verified-color-lvl1;
-          padding: ($baseline/10);
+        p {
+          color: $lighter-base-font-color;
+          font-style: italic;
+          letter-spacing: 1px;
+          text-align: center;
         }
+      }
 
-        // course enrollment status message
-        .sts-enrollment {
-          .label {
-            @extend %text-sr;
-          }
+      .course-status-completed {
+        background: $gray-l3;
+        color: $very-light-text;
 
-          .deco-graphic {
-            @extend %ui-depth3;
-            width: 40px;
-            position: absolute;
-            top: -4px;
-            right: -6px;
-          }
+        p {
+          color: #222;
 
-          // status message
-          .sts-enrollment-value {
-            background: $verified-color-lvl1;
+          span {
+            font-weight: bold;
           }
         }
       }
 
-      // CASE: "enrolled as" status - honor code
-      &.honor {
-
-        // changes to cover
-        .cover {
-          border-color: $honorcode-color-lvl2;
-          padding: ($baseline/10);
-        }
-
-        // status message
-        .sts-enrollment-value {
-          background: $honorcode-color-lvl1;
-        }
-      }
+      .enter-course {
+        @include button(simple, $button-color);
```

Comment:
oh man it would be so great to get rid of this old mixin, but i know it's probably out of scope.